azext_iot/_help.py

```diff
@@ -96,6 +96,39 @@
         az iot hub monitor-feedback -n {iothub_name} -d {device_id} -w {message_id}
 """
 
+helps[
+    "iot hub connection-string"
+] = """
+    type: group
+    short-summary: Manage IoT hub\'s connection string.
+"""
+
+helps[
+    "iot hub connection-string show"
+] = """
+    type: command
+    short-summary: Show the connection strings for an IoT hub.
+    examples:
+    - name: Show the connection strings for all IoT hubs in a subscription.
+      text: >
+          az iot hub connection-string show
+    - name: Show the connection strings for all IoT hubs in a resource group.
+      text: >
+          az iot hub connection-string show --resource-group MyResourceGroup
+    - name: Show all the connection string of an IoT hub using default key.
+      text: >
+          az iot hub connection-string show -n MyIotHub --all
+    - name: Show the connection string of an IoT hub using default policy and primary key.
```

Comment:
Because this becomes rendered for end users, can we be consistent with the capitalization of 'IoT Hub'?  In some cases, like this line its 'IoT **h**ub'.

Reply:
Fixed